THESIS.md

```diff
@@ -64,10 +66,14 @@ From Razzaq et al.'s work, we extracted and classified a list of measurable DevX
 
 #### Issues in Project Estimation
 
-Project estimation is another area where significant mistakes are commonly made. Many organizations rely on outdated or inaccurate data, often using tools like Gantt charts or relying on historical data that doesn't match the current project's context. This leads to faulty project estimations, especially when new projects differ substantially from past experiences. Project estimation models, such as the use of story points or task count statistics, attempt to predict project volume and complexity. While helpful to some extent, these methods are often flawed when the data they are based on is corrupted or misaligned with the project's unique requirements.
+Project estimation is another area where significant mistakes are commonly made. Many organizations rely on outdated or inaccurate data, often using tools like Gantt charts or relying on historical data that doesn't match the current project's context. This leads to faulty project estimations, especially when new projects differ substantially from past experiences. Project estimation models, such as the use of story points or task count statistics, attempt to predict project volume and complexity. While helpful to some extent, these methods are often flawed when the data they are based on is corrupted or misaligned with the project's unique context. [22]
 
 Another popular method, COCOMO 2, provides a more structured approach to project estimation but comes with its own set of limitations. Specifically, COCOMO 2 struggles with accurately reflecting team dynamics, work culture, and atmosphere. It also uses a broad set of parameters, some of which may be unclear or difficult to quantify. The rigid structure of models like COCOMO 2 can lead to inaccuracies, particularly in cases where human factors like satisfaction and team cohesion play significant roles in project outcomes.
 
+Techniques for estimating efforts are divided into non-algorithmic and algorithmic, only some of each group are described above. For example, the methods using story points as an indicator are based on expert assessment and are non-algorithmic, in turn COCOMO 2 is based on a clear algorithm with mathematical equations requiring specific quantitative data and coefficients. [5, 22]
+
+In this article, we will discuss the pros and cons of each approach, and outline a balanced strategy that will allow you to select relevant factors, indicators, an evaluation model or a hybrid of several models to achieve adequate results of estimating project efforts.
+
 #### A Correct Approach to Productivity and Project Estimation
```

Comment:
Everything after it, I mean from chapter "A correct approach to productivity and project estimation" should go to methodology section, because in methodology we should explain what we are doing..